inclusion/language.go

```diff
@@ -22,14 +22,14 @@ var conductLinks = "\nIn case of doubts please check our <https://bcneng.org/coc
-	{Filter: "these guys", Reply: "Instead of *guys*, perhaps you mean *gang*?... *[Please consider editing your message so it's more inclusive]*"},
-	{Filter: "my guys", Reply: "Instead of *guys*, perhaps you mean *crew*?... *[Please consider editing your message so it's more inclusive]*"},
-	{Filter: "those guys", Reply: "Instead of *guys*, perhaps you mean *people*?... *[Please consider editing your message so it's more inclusive]*"},
-	{Filter: "hey guys", Reply: "Instead of *guys*, perhaps you mean *y'all*?... *[Please consider editing your message so it's more inclusive]*"},
-	{Filter: "hi guys", Reply: "Instead of *guys*, perhaps you mean *everyone*?... *[Please consider editing your message so it's more inclusive]*"},
-	{Filter: "the guys", Reply: "Instead of *guys*, perhaps you mean *folks*?... *[Please consider editing your message so it's more inclusive]*"},
-	{Filter: "guys", Reply: "Instead of *guys*, have you considered a more gender-neutral pronoun like *folks*? *[Please consider editing your message so it's more inclusive]*"},
+	{Filter: "you guys", Reply: "Instead of *guys*, perhaps you mean *pals*? You can read more information about it at https://www.dictionary.com/e/you-guys/... *[Please consider editing your message so it's more inclusive]*"},
+	{Filter: "these guys", Reply: "Instead of *guys*, perhaps you mean *gang*? You can read more information about it at https://www.dictionary.com/e/you-guys/... *[Please consider editing your message so it's more inclusive]*"},
+	{Filter: "my guys", Reply: "Instead of *guys*, perhaps you mean *crew*? You can read more information about it at https://www.dictionary.com/e/you-guys/... *[Please consider editing your message so it's more inclusive]*"},
+	{Filter: "those guys", Reply: "Instead of *guys*, perhaps you mean *people*? You can read more information about it at https://www.dictionary.com/e/you-guys/... *[Please consider editing your message so it's more inclusive]*"},
+	{Filter: "hey guys", Reply: "Instead of *guys*, perhaps you mean *y'all*? You can read more information about it at https://www.dictionary.com/e/you-guys/... *[Please consider editing your message so it's more inclusive]*"},
+	{Filter: "hi guys", Reply: "Instead of *guys*, perhaps you mean *everyone*? You can read more information about it at https://www.dictionary.com/e/you-guys/... *[Please consider editing your message so it's more inclusive]*"},
+	{Filter: "the guys", Reply: "Instead of *guys*, perhaps you mean *folks*? You can read more information about it at https://www.dictionary.com/e/you-guys/... *[Please consider editing your message so it's more inclusive]*"},
+	{Filter: "guys", Reply: "Instead of *guys*, have you considered a more gender-neutral pronoun like *folks*? You can read more information about it at https://www.dictionary.com/e/you-guys/ *[Please consider editing your message so it's more inclusive]*"},
```

Comment:
```suggestion
	{Filter: "guys", Reply: "Instead of *guys*, have you considered a more gender-neutral pronoun like *folks*? You can read more information about it at https://www.dictionary.com/e/you-guys/... *[Please consider editing your message so it's more inclusive]*"},
```